frontend/src/components/Loadable/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { lazy, ComponentType } from "react";
+
+function Loadable<T>(importPath: {
```

Comment:
Just wanted to know whats the benefit of using this wrapper? We could also used the lazy function directly in the code.

Reply:
Yes you are correct but I have different thoughts over this one

1. we could also integrate retry functionality in the future (as the promise might fail also)
1. make a function to make unit test out of it (that too in future)